community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf

```diff
@@ -117,14 +117,9 @@ variable "tags" {
 }
 
 variable "disk_type" {
-  description = "Boot disk type, can be either pd-ssd, pd-standard, pd-balanced, or pd-extreme."
+  description = "Boot disk type, can be either hyperdisk-balanced, hyperdisk-extreme, pd-ssd, pd-standard, pd-balanced, or pd-extreme."
```

Comment:
Can you make these same changes to login and controller?

Reply:
Done.